quickstart/201-mysql-fs-db/main.tf

```diff
@@ -20,21 +25,15 @@ resource "random_password" "password" {
   override_special = "_"
 }
 
-// Manages the Resource Group where the resource exists
-resource "azurerm_resource_group" "default" {
-  name     = "mysqlfsRG-${random_pet.rg_name.id}"
-  location = var.location
-}
-
-// Manages the Virtual Network
+# Manages the Virtual Network
 resource "azurerm_virtual_network" "default" {
   name                = "vnet-${random_string.name.result}"
   location            = azurerm_resource_group.default.location
   resource_group_name = azurerm_resource_group.default.name
```

Comment:
Since the `azurerm_resource_group.default` has been removed, I guess we should change this line to:

```hcl
resource_group_name           = azurerm_resource_group.rg.name
```